mcp-registry.json

```diff
@@ -0,0 +1,336 @@
+{
+  "filesystem": {
+    "id": "filesystem",
+    "name": "Filesystem",
+    "description": "Secure file operations with configurable access controls",
+    "category": "Development",
+    "config": {
+      "type": "stdio",
+      "command": "npx",
+      "args": ["-y", "@modelcontextprotocol/server-filesystem", "."],
+      "connectionMode": "strict"
+    },
+    "setupInstructions": "No setup required. Specify directory path as argument.",
+    "tags": ["files", "local", "development", "essential"]
+  },
+  "git": {
+    "id": "git",
+    "name": "Git",
+    "description": "Tools to read, search, and manipulate Git repositories",
+    "category": "Development",
+    "config": {
+      "type": "stdio",
+      "command": "uvx",
+      "args": ["mcp-server-git", "--repository", "."],
```

Comment:
doesn't exist

[ 10:14AM ]  [ karaj@MacBook-Pro:~ ]
 $ uvx mcp-server-git --repository .
Installed 30 packages in 64ms
ERROR:mcp_server_git.server:. is not a valid Git repository
[ 10:14AM ]  [ karaj@MacBook-Pro:~ ]